js/script.js

```diff
@@ -332,3 +332,427 @@
+//   this.age = age;
+//   this.eyeColor = eye;
+// }
+
+// let myFather = new Person("Стрелюков", "Михаил", 56, "Голубой");
+
+// console.log("Моему отцу " + myFather.age + ".");
+
+// function ucFirst(str) {
+//   if (!str) return str;
+
+//   return str[0].toUpperCase() + str.slice(1);
+// }
+
+// alert(ucFirst("вася"));
+
+// let style = [" Джаз" , " Блюз"];
+
+// style.push (" Рок-н-ролл");
+
+// style[1] = " Классика";
+// alert(style);
+
+// alert(style.shift([1]));
+
+// style.unshift(" Реп", " Регги");
+// alert(style);
+
+// function unique(arr) {
+//   return Array.from(new Set(arr));
+// }
+
+// let values = ["Hare", "Krishna", "Hare", "Krishna",
+//   "Krishna", "Krishna", "Hare", "Hare", ":-O"
+// ];
+
+// alert( unique(values) ); // Hare,Krishna,:-O
+
+// let salaries = {
+//   "John": 100,
+//   "Pete": 300,
+//   "Mary": 250,
+// };
+
+// function sumSalaries(salaries) {
+//   let sum = 0;
+//   for (let salary of Object.values(salaries)) {
+//     sum += salary;
+//   }
+
+//   return sum;
+// }
+
+// alert(sumSalaries(salaries));
+
+// let user = {
+//   name: 'John',
+//   age: 30
+//   };
+
+// function count(obj) {
+//   return Object.keys(obj).length;
+// }
+
+// console.log(count(user));
+
+// let arr = ["Vasya", "Petrovich"];
+
+// let [firstName, surname] = arr;
+
+// alert(firstName);
+
+//task 1;
+// const func = (a, b) => a === b;
+
+// console.log(func(2, 2));
+
+//task 2;
+// const func = (a, b) => a + b > 10;
+
+// console.log(func(2, 2));
+
+//task 3;
+// const func = (a) => a < 0;
+
+// console.log(func(-1));
+
+//task 4;
+// let str = '';
+
+// for(let i = 1; i <=9 ; i++) {
+//   str = str + i;
+// }
+
+// console.log(str);
+
+//task 5;
+// let str = "";
+
+// for(i = 0; i < 20; i++) {
+// console.log(str += "x")
+// }
+
+// task 6;
+// for (let i = 1; i <= 9; i++) {
+//   let str = '';
+
+//   for (let j = 1; j <= i; j++) {
+//     str = str + i;
+//   }
+//   console.log(str);
+// }
+
+// task 7;
+// let str = "";
+
+// for(i = 0; i<5; i++){
+//   console.log(str += "xx")
+// }
+
+// task 8;
+// for(i = 1; i <= 9; i++) {
+//   str = '';
+
+//   for(j = 1; j <= 3; j++){
+//     str = str + i;
+//   }
+//   console.log(str);
+// }
+
+// Array tasks 1;
+// const array1 =['a', 'b', 'c'];
+// const array2 = [1, 2, 3];
+// console.log(array1.concat(array2));
+
+// const array3 = [...array1, ...array2];
+// console.log(array3);
+
+// Array tasks 2;
+// const array = ['a', 'b', 'c'];
+// array.push(1, 2, 3);
+
+// console.log(array);
+
+// Array tasks 3;
+// const array = [1, 2, 3, 4, 5];
+// array.splice(1, 2);
+
+// console.log(array);
+
+// Array tasks 4;
+// const array = [1, 2, 3, 4, 5];
+// const removed = array.splice(1, 3);
+
+// console.log(removed);
+
+// Array tasks 5;
+// const array = [1, 2, 3, 4, 5];
+// array.splice(2, 0, 'a', 'b', 'c');
+
+// console.log(array);
+
+// Array tasks 6;
+// const array = [1, 2, 3, 4, 5];
+// array.splice(1, 0, 'a', 'b'), array.splice(6, 0, 'c'), array.splice(8, 0, 'e');
```

Comment:
я была не права, в одну строчку выполнить нельзя 
```suggestion
// array.splice(1, 0, 'a', 'b');
array.splice(6, 0, 'c');
array.splice(8, 0, 'e');
```